File: PRAK603-2210817120013-Fathiah_Nuraisyah_Radam.py
baris, kolom = map(int, input().split())
if (baris != kolom):
	print("Jumlah tidak sama")
else:
	matrix1 = list(map(int, input().split()))
	matrix2 = list(map(int, input().split()))
	for i in range(baris):
		print(matrix1[i] * matrix2[i], end=" ")
	print()

File: PRAK603-2210817120013-Fathiah_Nuraisyah_Radam.c
#include <stdio.h>
int main(){
      int baris1, baris2;
      scanf("%d %d", &baris1, &baris2);
      if (baris1 != baris2){
            printf("Jumlah tidak sama");
            return 0;
      }
      int a[baris1];
      int b[baris2];
      int i;
            for (i = 0; i < baris1; i++){
                  scanf("%d", &a[i]);
            }
            for (i = 0; i < baris2; i++){
                  scanf("%d", &b[i]);
            }
            for (i = 0; i < baris1; i++){
                  printf("%d ", a[i] * b[i]);
            }
            return 0;
}